common/djangoapps/student/views.py

```diff
@@ -1167,6 +1167,8 @@ def login_user(request, error=""):  # pylint: disable-msg=too-many-statements,un
 @social_utils.strategy("social:complete")
 def login_oauth_token(request, backend):
     """
+    DEPRECATED - Note that this endpoint is deprecated.  Please use AccessTokenExchangeView instead.
```

Comment:
I have changed the code to use `warnings.warn`.  For performance reasons, they are turned off on Prod.  They may even be turned off by default on devstack for the time being because of multitude xBlock warnings.  However, they are useful to developers who want to catch these.

There is also a 'log once' mode that could be used.